### ISSUE TITLE
BIT does not find secodn SSD as backup destination

### ISSUE THREAD

**Issue by Osmandfan** [bit-team/backintime#1826]
![image](https://github.com/user-attachments/assets/a09b717c-9384-42a8-993b-839bf9fdf629)

Installed back in time  from standard repository in Linux Mint.

I have a second harddrive installed in my laptop. BIT does not find this although it is automatically mounted and formatted as EXT4.

> --  diagnose give an error message.

see attachment.

**Comment by buhtz** [comment]
Hello Osmandfan,
thank you for reporting the issue. Let's see if I can help you.

- Please let me know which version of Back In Time you are using.
- Please post the output of `backintime --debug`.
- Can you find the second harddrive with your regular file explorer outside form BIT?

I also don't see an error message in your screenshot. Everything looks fine.

Please provide more details to make clear what the problem is. Otherwise I can not help you.

Best
Christian

**Comment by Osmandfan** [comment]
Thank you very much for your quick response.

1. BIT version 1.2.1

2. debug DEBUG: [common/backintime.py:583 argParse] Arguments: {'debug': True} | unknownArgs: []

Back In Time
Version: 1.2.1

Back In Time comes with ABSOLUTELY NO WARRANTY.
This is free software, and you are welcome to redistribute it
under certain conditions; type `backintime --license' for details.

DEBUG: [common/backintime.py:670 getConfig] config file: /home/hart/.config/backintime/config
DEBUG: [common/backintime.py:671 getConfig] share path: /home/hart/.local/share/backintime
DEBUG: [common/backintime.py:672 getConfig] profiles: 1=Hoofdprofiel
3 . Yes I have no problem finding the drive. It is on my desktop 
![afbeelding](https://github.com/user-attachments/assets/563a92e3-0976-4f56-ac21-2b6af35f1a89)
It is the backup folder on the DATA partition
![afbeelding](https://github.com/user-attachments/assets/d805ca14-d75a-4da4-969b-9e32e996fb1b)

**Comment by buhtz** [comment]
Thank you for reporting back. I still don't understand the problem. I don't see a problem or error message in your report.

**Comment by Osmandfan** [comment]
Thank you buhtz I cannot pick the backup folder on the data HDD. These are my options. I can only choose a folder within my personal home folder. Which makes no sense to me. Guess I am missing something.
![afbeelding](https://github.com/user-attachments/assets/a0d6eca5-4241-4cdc-874a-bebe21186fb3)

**Comment by buhtz** [comment]
Your screenshot looks fine. I see no error and still don't understand the problem.
Please describe the steps you do and what doesn't behave as you expect.

**Comment by Osmandfan** [comment]
I can only choose a folder within my personal home folder. Which makes no sense to me. Guess I am missing something. I can not navigate to the backup folder on the other SSD.

**Comment by buhtz** [comment]
Are you talking about this window?
![image](https://github.com/user-attachments/assets/baea96fe-1936-43bd-a488-e88c7e476127)

This is a regular folder/file selection dialog provided by your Operating System (or Desktop Environment) and not Back In Time itself. Maybe you are not familiar with it and don't know how to use it?

You see three arrows right from the path widget. Click on the 3rd arrow that points to the top. This will bring you one level up in the folder hirarchie.

**Comment by emtiu** [comment]
You can also click on "Computer" on the left there, and then goto `/mnt/Timeshift`, or wherever you want to go.

**Comment by Osmandfan** [comment]
Yes thank you I know. I can navigate upwards  but I can't switch to choosing the second HDD it does not show up when I navigate upwards. It stay on the first Drive.

**Comment by buhtz** [comment]
Can you show us a screenshot please what you see after clicking on "Computer".

**Comment by Osmandfan** [comment]
![afbeelding](https://github.com/user-attachments/assets/e76b183a-3f4d-4243-9128-08eabcdf7394)

**Comment by buhtz** [comment]
Sorry for asking. You might be new to GNU/Linux? There is only one "fileystem". You never see devices (hard disks, usb sticks) in diaglos like this like you do see in MS Windows.

Go to the folder /mnt/Daten. That is your second hard drive. Look for the folder not the drive.

Klick on / . This is the root, the top most "folder", of your filesystem. Then klick on "mnt". And then "Daten".

**Comment by Osmandfan** [comment]
It stays on this SSD. 
![afbeelding](https://github.com/user-attachments/assets/b03c6993-a67a-4934-901e-927dbaafde05)
can not pick a folder on the second drive
which is this one 
![afbeelding](https://github.com/user-attachments/assets/783df6f9-0c92-49f0-ad00-8e2975e860d4)

**Comment by Osmandfan** [comment]
I indeed did not know you can't navigate like you can in windows. Yes I am new to linux only used it for a years or so. Still learning the basics.
I managed to copy and past the folders path 
Thank you very much! It works.

![afbeelding](https://github.com/user-attachments/assets/0cf1f2f9-758b-4ff7-9f1d-46e5103708e1)

**Comment by Osmandfan** [comment]
Will a make incremental backups? Only changed files?

**Comment by buhtz** [comment]
> Will a make incremental backups? Only changed files?

Every backup is a full backup. But only changed files are copied. The unchanged files are hard linked. See this FAQ about the details: https://github.com/bit-team/backintime/blob/dev/FAQ.md#does-back-in-time-create-incremental-or-full-backups

About the folder selction: Maybe this help you to understand the process?
![Peek 2024-08-03 17-15](https://github.com/user-attachments/assets/64486c28-e1b9-4fd5-9bba-2e0f5bdbda63)

**Comment by Osmandfan** [comment]
Yes that helps a lot. Thank you very much. Did not know it was a subfolder of mnt.
Thank you also for the link. Have to learn a lot. :-)
Very happy this works Just lost 8 months of work due to a failed timeshift restore .  Want to be sure that does never ever happen again. Tried everything to recover the deleted files in the home folder but all attempts stranded. There is no decent unerase program for linux I understand.

**Comment by emtiu** [comment]
I'm sorry you lost so much data. Keeping good backups is a learning process, and even experienced users sometimes still suffer failures. I wish you more success with Back In Time :)

**Comment by Osmandfan** [comment]
How thoughtful of you emtiu. When I switched to Linux I spent weeks on the linux forum trying to find out what is the best backup strategy. Came up with a combination of foxclone, timeshift and NAS sync. Despite that for unknown reasons part of the personal folder got deleted after timeshift restore action. Although I  do **not** have home folder included in timeshift. Still a mystery to me how that got deleted. But very nice of you to care. Wishing you a good weekend!
